query/src/sql/sql_parser.rs

```diff
@@ -552,4 +586,35 @@ impl<'a> DfParser<'a> {
             false
         }
     }
+
+    fn replace_show_database(
```

Comment:
If we don't use `SELECT name AS Database FROM ...`, we can remove it

Reply:
> If we don't use `SELECT name AS Database FROM ...`, we can remove it

For `SHOW DATABASES WHERE Database = 'ss' `, we will transfer it to sql like following,
`SELECT name FROM system.databases WHERE name = 'ss' ORDER BY name`.

In this transform, we need replace `Database` to `name`, so I add `replace_show_database`  function.

If database support alias in where clause , we can transfer to `SELECT name  AS Database FROM system.databases WHERE Database = 'ss' ORDER BY Database`,  so `replace` is unnecessary , and we can  remove `replace_show_database` function.

Reply:
When we use database to filter name, it is always confusing
```
mysql> SHOW DATABASES WHERE Database = 'default';
+---------+
| name    |
+---------+
| default |
+---------+
```

Reply:
We can fix this problem in two ways:
- Remove replace_show_database method
``` text
mysql> SHOW DATABASES WHERE name = 'default';
+---------+
| name    |
+---------+
| default |
+---------+
```
- Change `SELECT name FROM system.databases WHERE name = {}` to  `SELECT name AS Database FROM system.databases WHERE name = {}`
``` text
mysql> SHOW DATABASES WHERE Database = 'default';
+----------+
| Database |
+----------+
| default  |
+----------+
```

Reply:
Got it. But If we choose the first one, it means we will not support sql like 
> `SHOW DATABASES WHERE Database = 'default';`

is it ok?